Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```diff
@@ -5065,6 +5106,7 @@ script:
       type: String
   dockerimage: demisto/py3-tools:1.0.0.117220
   isfetch: true
+  isfetch:marketplacev2: false
```

Comment:
This is what should remove all the mirroring fields? is it working?

Reply:
The YML fields related to mirroring are:
* **`ismappable`** - determines if the remote schema can be retrieved (implemented in the `get-mapping-fields` command)
* **`isremotesyncin`** -  determines if mirroring from the integration to XSOAR is supported.
* **`isremotesyncout`** - determines if mirroring from XSOAR to the integration is supported.

As we previously discussed, disabling these on marketplacev2 is currently _not_ supported in the content item schema in the SDK.

`isfetch:marketplacev2: false` only disables the ability to choose whether to fetch incidents (alerts on XSIAM).